tests/test_labview231_header.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from silverlabnwb.header import LabViewHeader231, LabViewVersions, Modes
+
+
+def do_header231_test(lab_view_header, expected):
+    header = LabViewHeader231.from_file(lab_view_header)
```

Comment:
This should be `LabViewHeader.from_file(...)`. 

It shouldn't make a difference with the code the way is at the moment, but in general we should be checking that we get the right version even if we call the `from_file` method on the base class.